expression/builtin_time_vec.go

```diff
@@ -2458,11 +2458,91 @@ func (b *builtinUTCTimestampWithoutArgSig) vecEvalTime(input *chunk.Chunk, resul
 }
 
 func (b *builtinConvertTzSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinConvertTzSig) vecEvalTime(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+
+	// fromTz
+	fromTzBuf, err := b.bufAllocator.get(types.ETString, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(fromTzBuf)
+	errFromTz := b.args[1].VecEvalString(b.ctx, input, fromTzBuf)
+
+	// toTz
+	toTzBuf, err := b.bufAllocator.get(types.ETString, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(toTzBuf)
+	errToTz := b.args[2].VecEvalString(b.ctx, input, toTzBuf)
+
+	// dt
+	errDt := b.args[0].VecEvalTime(b.ctx, input, result)
+
+	if errFromTz != nil || errToTz != nil || errDt != nil {
+		result.SetNulls(0, n, true)
```

Comment:
Should we call the row-based function instead of setting all nulls? Because we don't know which row generates the error.